tests/data/expected/main/openapi/nullable_31.py

```diff
@@ -0,0 +1,17 @@
+# generated by datamodel-codegen:
+#   filename:  nullable_31.yaml
+#   timestamp: 2019-07-26T00:00:00+00:00
+
+from __future__ import annotations
+
+from typing import List
+
+from pydantic import BaseModel
+
+
+class Apple(BaseModel):
+    pass
+
+
+class Basket(BaseModel):
+    apples: List[Apple] | None
```

Comment:
previously, before the change in this PR, this would have been:

```suggestion
    apples: List[Apple]
```